src/main/java/org/sciborgs1155/robot/Robot.java

```diff
@@ -38,12 +43,18 @@ public class Robot extends CommandRobot implements Logged {
   private final CommandXboxController driver = new CommandXboxController(OI.DRIVER);
 
   // SUBSYSTEMS
-  private final Drive drive = Drive.create();
+  @Log.NT private final Drive drive = Drive.create();
+
+  @Log.NT private final Shooter shooter = Shooter.create();
```

Comment:
can you add a `RobotType` enum to `Constants` similar to last year's code
the usage here would be
```java
@Log.NT private final Shooter shooter = switch (Constants.RobotType) {
    case HOMEWORK -> Shooter.createNone(); // creates a nonexistent shooter for a chassis robot
    case DISCOVERY -> Shooter.create();
}
```
it would also involve `NoPivot` `NoShooter` and `NoFeeder`, which should be simple enough